_static/.dummy

```diff
@@ -0,0 +1 @@
+To keep this directory in the git repository.
```

Comment:
why do you want to track an empty directory?

Reply:
Otherwise, a Sphinx warning tells me there is no `_static` directory.

Reply:
So I added another sentence saying it's for Sphinx.